### PR TITLE
Added 2 security bulletins

### PR DESCRIPTION
**Pull request by cloudmaniac**
## Describe the Change

This PR adds 2 security bulletins

## Review Changes

💻 [Add Preview URL]()

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1275#" title="Author self-approved">cloudmaniac</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/security/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @cloudmaniac. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/09.5-security/20-security-bulletins.md
Comment (karl-cardenas-coding) [inline content/docs/09.5-security/20-security-bulletins.md:37]:
```suggestion
A local user with permission to edit files can use this flaw to change a file not permitted by the security policy, resulting in privilege escalation.
```

File: content/docs/09.5-security/20-security-bulletins.md
Comment (karl-cardenas-coding) [inline content/docs/09.5-security/20-security-bulletins.md:43]:
```suggestion
* For Palette SaaS, this has been addressed and requires no user action. 
```

File: content/docs/09.5-security/20-security-bulletins.md
Comment (karl-cardenas-coding) [inline content/docs/09.5-security/20-security-bulletins.md:44]:
```suggestion
* For ​​Palette self-hosted deployments, please upgrade to newer versions greater than or equal to  v2.6.70 to address the reported vulnerability.
```

File: content/docs/09.5-security/20-security-bulletins.md
Comment (karl-cardenas-coding) [inline content/docs/09.5-security/20-security-bulletins.md:61]:
```suggestion
On May 3 2022, OpenSSL published a security advisory disclosing a command injection vulnerability in the `c_rehash` script included with the OpenSSL library. Some operating systems automatically execute this script as a part of normal operations, which could allow an attacker to execute arbitrary commands with elevated privileges.
```

File: content/docs/09.5-security/20-security-bulletins.md
Comment (karl-cardenas-coding) [inline content/docs/09.5-security/20-security-bulletins.md:63]:
```suggestion
Palette is not directly affected by this vulnerability. However, if your cluster profile is using the vSphere CSI pack, version v2.3 or below, it contains a vulnerable version of the `c_rehash` script.
```

File: content/docs/09.5-security/20-security-bulletins.md
Comment (karl-cardenas-coding) [inline content/docs/09.5-security/20-security-bulletins.md:65]:
```suggestion
```

File: content/docs/09.5-security/20-security-bulletins.md
Comment (karl-cardenas-coding) [inline content/docs/09.5-security/20-security-bulletins.md:71]:
```suggestion
The `c_rehash` script does not sanitize shell metacharacters properly to prevent command injection. This script is distributed by some operating systems, and by extension, in container images, in a manner where it is automatically executed. On such operating systems, an attacker could execute arbitrary commands with the privileges of the script.
```

File: content/docs/09.5-security/20-security-bulletins.md
Comment (karl-cardenas-coding) [inline content/docs/09.5-security/20-security-bulletins.md:77]:
```suggestion
This vulnerability has been addressed in the vSphere CSI pack greater than or equal to version v2.6.
```

File: content/docs/09.5-security/20-security-bulletins.md
Comment (karl-cardenas-coding) [inline content/docs/09.5-security/20-security-bulletins.md:83]:
```suggestion
Update cluster profiles using the vSphere CSI pack to version v2.6 or greater. Apply the updated cluster profile changes to all clusters consuming the cluster profile.
```

**Review by karl-cardenas-coding — APPROVED** [review]
Loooks good, just some minor suggestions.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1275#" title="Author self-approved">cloudmaniac</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1275#pullrequestreview-1411845241" title="Approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/security/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
